docs/dictionary/function/specialFolderPath.lcdoc

```diff
@@ -102,8 +102,13 @@ The following <folderIdentifier> values are supported:
   * 0x000a: The "Recycle Bin".
 
 
-**On Mac OS systems**, the following <folderIdentifier> values 
-are supported: 
+**On Mac OS systems**, when there are no user stacks loaded into memory or
```

Comment:
We need to check if this only affects the macOS platform or if it also affects Windows and Linux environments also.

Reply:
@seaniepie yes it will be the same on all platforms. When there are no user stacks one of the IDE stacks will be `the defaultStack` so `specialFolderPath("resources")` will be relative to that. So probably this should be a non-platform specific note.